osu.Game/Configuration/OsuConfigManager.cs

```diff
@@ -171,9 +168,23 @@ protected override void InitialiseDefaults()
             Set(OsuConfig.CanForceOptimusCompatibility, true);
             Set(OsuConfig.ConfineMouse, Get<bool>(OsuConfig.ConfineMouseToFullscreen) ?
                 ConfineMouseMode.Fullscreen : ConfineMouseMode.Never);
+
+            GetBindable<bool>(OsuConfig.SavePassword).ValueChanged += delegate { eventPassword(); };
+            GetBindable<bool>(OsuConfig.SaveUsername).ValueChanged += delegate { eventUsername(); };
 #pragma warning restore CS0612 // Type or member is obsolete
         }
 
+        private void eventPassword()
+        {
+            if ((GetBindable<bool>(OsuConfig.SavePassword) == true) && (GetBindable<bool>(OsuConfig.SaveUsername) == false))

```

Comment:
`if Get<bool>(OsuConfig.SavePassword) && ...`

no need for == true and no need to get the bindable.